lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -41,7 +33,19 @@
         padding: $base-spacing;
       }
     }
+  }
+
+  .card__copy {
+    &.-share {
+      h3:last-of-type,
```

Comment:
We seem to have a decent amount of situations where we want things to hide/appear on mobile (such as homepage headers) that don't really make sense to make specific CSS rules for each scenario. Perhaps something to merge in for a quick fix now, and then refactor at the Neue level in the near future.